test/test_infra.py

```diff
@@ -42,7 +43,7 @@ class InfrastructureTests(unittest.TestCase):
 
     def test_quote_backslashes(self):
         # ensure we're not using unneeded backslash to escape strings
-        issues = []
+        issues : typing.List[typing.Tuple[str, int, str]] = []
```

Comment:
I thought it was odd that Flake8 didn't complain about the [PEP8 pet peeve](https://www.python.org/dev/peps/pep-0008/#pet-peeves) of whitespace before a colon, but I see we have `E203` ignored in our `pyproject.toml`. Odd -- not sure why. Can we please un-ignore that and fix these? From a quick check, the only instances are the ones in this file. Needs to be done in the tool.autopep8 and tool.flake8 section.

Reply:
> I thought it was odd that Flake8 didn't complain about the [PEP8 pet peeve](https://www.python.org/dev/peps/pep-0008/#pet-peeves) of whitespace before a colon, but I see we have `E203` ignored in our `pyproject.toml`. Odd -- not sure why. Can we please un-ignore that and fix these? From a quick check, the only instances are the ones in this file. Needs to be done in the tool.autopep8 and tool.flake8 section.

Good, I don't like that either and was just relying on the tooling :)